openerp/addons/base/res/res_partner.py

```diff
@@ -68,6 +85,11 @@ def fields_view_get_address(self, cr, uid, arch, context={}):
                 doc = etree.fromstring(arch)
                 for node in doc.xpath("//div[@class='address_format']"):
                     tree = etree.fromstring(v)
+                    # needs to apply modifiers
+                    for childnode in tree.getchildren():
+                        modifiers = {}
+                        orm.transfer_node_to_modifiers(childnode, modifiers)
```

Comment:
Because `doc` object is already parsed so `modifiers` attribute should be present at the end of this method.

I guess this method is called to late, but I don't know how to fix this in that way!